src/widgets/PrescriptionInfo.js

```diff
@@ -82,7 +82,7 @@ const PrescriptionInfoComponent = ({
             )}
           </FlexView>
 
-          <FlexRow flex={1} justifyContent="">
+          <FlexRow flex={1}>
```

Comment:
fixing #3541

Reply:
thanks - testing this PR was tricky without this fix :)